### DISCUSSION TITLE
Save color scheme

### DISCUSSION THREAD

**Discussion by Stuntflyer** [mikeberger/borg_calendar#211] [General]
I tried installing the calendar on another computer using my saved preferences and the backed up .ics file. The color scheme I created did not install. How would I back that up?

Mike

**Comment by mikeberger** [comment]
The color schemes are one of the very few preferences that are stored in the database.

To copy the database to another computer, use Export XML to create a backup. (I should rename that menu item to export database). then on the other machine do: Import entire backup file - into an empty database.

ICS files should be used to create a backup that can be imported into another calendar program.

**Reply by Stuntflyer** [reply]
Thanks!
